configs/la-metro-dashboard.production.conf.nginx

```diff
@@ -0,0 +1,68 @@
+server {
+    listen 443;
+
+    # Set the domain name that Nginx will listen on. If the app needs to listen on
+    # additional domains (such as the www subdomain of the root domain), add them
+    # here as a space-delimited list, and adjust the second server block below similarly.
+    server_name la-metro-dashboard.datamade.us;
```

Comment:
N.b., I am transitioning `la-metro-dashboard.datamade.us` to production, and using `la-metro-dashboard-staging.datamade.us` for staging.